### DISCUSSION TITLE
Page selector of Search Results not visible

### DISCUSSION THREAD

**Discussion by CasmahaOGD** [microsoft-search/pnp-modern-search#3846] [Q&A]
**Version used**
4.12.2

**Describe the bug**
One user does not see the page selector of the Search Results web part, user was removed from the site collection and associated groups, and added again but this did not work. Everything browser related was tried aswell but to no avail.
Everything else in the site collection is working correctly, Filters webpart, Searchbox webpart etc.
(there is also enough data searchable and user has access to all of this to even get a page selector. More than 100 items should be visible).

**To Reproduce**
Not able to reproduce this ourselves

**Expected behavior**
Able to see and use the page selector in the Search Results web part

**Screenshots**
Not able to add because of company data, but it is just the Search Results web part without the page selector at the bottom.

**Desktop (please complete the following information):**
- Browser: Chrome
**Additional context**
None

**Comment by kasperbolarsen** [comment]
Please update to the current version as we can not support older versions

**Comment by CasmahaOGD** [comment]
Sorry, did not see the version as it was in the template, i am using the current version

**Comment by kasperbolarsen** [comment]
re "More than 100 items should be visible" , could you enable the "Show Results count" setting, and verify who many item the user has access to ?

**Comment by CasmahaOGD** [comment]
There are 126 results all of which the user has access to.

**Comment by wobba** [comment]
The page selector is displayed based on the value in the response saying how many results you have. That count is an estimate so the behavior is what it is. If the number is equal or less to the page size a paging control will not be shown.

**Reply by CasmahaOGD** [reply]
The user has access to the results as in the user has permissions for the results, they are not visible in the web part because only the first 13 results are displayed. Also this user is the only one with this issue out of 58 users.
